memory_core/genesis_new/mem.vp

```diff
@@ -72,21 +47,19 @@ assign sel_mask = {
 );
 
 /*
-
 // Instance of mem module
-TS1N16FFCLLSBLVTC128X64M4SW m (
+TS1N16FFCLLSBLVTC512X16M8S m (
   .CLK(clk), // Clock - 1
   .CEB(~cen), // Chip Enable (low) - 1
   .WEB(~wen), // Write Enable (low) - 1
   .A(addr), 
   .D(data_in),
-  .BWEB(sel_mask),
-  .Q(full_mem_out),
+  .BWEB(1'b1),
```

Comment:
Since the memory we're using does not have partial write, this port does not exist, so it should be removed

Reply:
addressed this in commit